extensions/GlobalMessages/GlobalMessagesF.i18n.php

```diff
@@ -215,6 +216,7 @@
 	'feed-unavailable' => 'This message is displayed when a user tries to use an RSS or Atom feed on a wiki where such feeds have been disabled.',
 	'feed-atom' => '{{optional}}',
 	'feed-rss' => '{{optional}}',
+	'file-resolution-exceeded' => 'Error dispayed when user tries to upload image with too high resolution',
```

Comment:
dispayed -> disp**l**ayed? (I don't know if it's engineers or the localization team that are supposed to correct this but leaving a comment just in case)